lib/service/customer_support_service.dart

```diff
@@ -147,13 +149,50 @@ class CustomerSupportServiceImpl extends CustomerSupportService {
         continue;
       }
     }
+    return issues;
+  }
+
+  @override
+  // this function is used to get the list of issues, and announcements,
+  // if announcements already in the list of issues, it will be removed
+  Future<List<ChatThread>> getChatThreads() async {
+    final issues = await _getIssues();
+    // add announcement
+    final announcement = injector<AnnouncementService>().getLocalAnnouncements()
+      ..removeWhere((element) {
+        final issueId = injector<AnnouncementService>()
+            .findIssueByAnnouncement(element.announcementContentId);
+        final issue =
```

Comment:
handle for case `issueId` is null

Reply:
we dont have to. If issueId is null, then condition element.issueId == issueId can not be true.